src/components/Header/MarketplaceHeader.tsx

```diff
@@ -43,7 +54,8 @@ const MarketplaceHeader: React.FC = () => {
         history.push('/marketplace/wishList')
         break
       case 'Inventory':
-        history.push('/inventory/home/draft')
+        if (!account) return popup({ success: false, message: t('connectWallet') })
+       history.push('/inventory/home/draft')
```

Comment:
if not connect to wallet then it should to redirect to marketplace becauseof user not connected and enventory have no data

Reply:
done